engine/apps/user_management/models/organization.py

```diff
@@ -303,6 +307,14 @@ def web_link_with_uuid(self):
         # It's a workaround to pass some unique identifier to the oncall gateway while proxying telegram requests
         return urljoin(self.grafana_url, f"a/grafana-oncall-app/?oncall-uuid={self.uuid}")
 
+    @property
+    def slack_is_configured(self) -> bool:
+        return self.slack_team_identity is not None
+
+    @property
+    def telegram_is_configured(self) -> bool:
+        return self.telegram_channel.count() > 0
```

Comment:
these are now needed for [this functionality](https://github.com/grafana/oncall/pull/3128/files#diff-9d96504027309f2bd1e95352bac1433b09b60eb4fafb611b52a6c15ed16cbc48R271-R272)